src/left-nav.jsx

```diff
@@ -283,7 +276,7 @@ const LeftNav = React.createClass({
   },
 
   _setPosition(translateX) {
-    const leftNav = ReactDOM.findDOMNode(this.refs.clickAwayableElement);
```

Comment:
Ohhh, thanks for catching. I tested on my phone and it wasn't working but I was clicking the button, not swiping. Now I get it.